backend/apis/clients/get-clients-by-id.api.js

```diff
@@ -0,0 +1,133 @@
+const {
+  app,
+  databaseError,
+  pool,
+  invalidRequest,
+  notFound,
+} = require("../../server");
+const mysql = require("mysql");
+const { checkValid, validId } = require("../utils/validation-utils");
+const {
+  responseFullName,
+  responseBoolean,
+  responseDateWithoutTime,
+} = require("../utils/transform-utils");
+const path = require("path");
+const fs = require("fs");
+const rawGetSql = fs.readFileSync(
+  path.join(__dirname, "./get-clients-by-id.sql")
+);
+
+app.get("/api/clients-by-id", (req, res) => {
+  const validationErrors = Array.isArray(req.query.clientId)
+    ? checkValid(req.query, validArray(validId("clientId")))
```

Comment:
validArray is not being imported, so this will throw an error. See line 9 where it should be imported.